src/pages/Accounts/AccountToken/AccountBalance.tsx

```diff
@@ -35,8 +37,12 @@ export const AccountBalance: React.FC<AccountBalanceProps> = ({
   backLabel,
 }) => {
   const navigate = useNavigate()
+  const translate = useTranslate()
   const asset = useAppSelector(state => selectAssetById(state, assetId))
   const opportunitiesFilter = useMemo(() => ({ assetId, accountId }), [assetId, accountId])
```

Comment:
ocd: can we rename this while we're here, this does nothing opportunities and I was turbo confused reading this :kek: